test/corpus/errors.txt

```diff
@@ -40,6 +40,31 @@ regex with incorrect modifiers
   (ERROR
     (identifier)))
 
+================================================================================
+PCRE2 invalid regex
+================================================================================
+begin
+  /\c\c/
```

Comment:
I'm not sure I understand the error with this example?

Reply:
There are non-printing control characters in the line that Github isn't showing. I'll try to find something that is more readable.

Reply:
It might be worth including a comment in the test to make this explicit.